index.bs

```diff
@@ -1767,6 +1771,9 @@ When this method is invoked, the user agent MUST execute the following algorithm
 
 1. If the length of <code>|pkOptions|.{{PublicKeyCredentialCreationOptions/user}}.{{PublicKeyCredentialUserEntity/id}}</code> is not between 1 and 64 bytes (inclusive) then throw a {{TypeError}}.
 
+1. If <code>|pkOptions|.{{PublicKeyCredentialCreationOptions/user}}.{{PublicKeyCredentialUserEntity/language}}</code> is present
```

Comment:
Should we require `language` and `direction` be present if the other one is? i.e. if `language` is present, `direction` cannot be undefined.

Reply:
I don't think so, String-Meta specifically describes these as separate values. It doesn't look like either should require the other.